### DISCUSSION TITLE
no environment-ufsreplay.yml and UFS_forcing_ensemble_generator.py in cloned SCM6.0.0

### DISCUSSION THREAD

**Discussion by AnningCheng-NOAA** [NCAR/ccpp-scm#389] [Cases, Initial Conditions, Forcing] [answered]
Hi, 

There no nvironment-ufsreplay.yml and UFS_forcing_ensemble_generator.py   in cloned SCM6.0.0 
git clone --recursive -b v6.0.0 https://github.com/NCAR/ccpp-scm
/scratch1/NCEPDEV/global/Anning.Cheng/ccpp-scm

Where and how can I have those file?

**Comment by AnningCheng-NOAA** [comment, marked as answer]
those files can be found only check out using 
git clone --recursive  https://github.com/NCAR/ccpp-scm

The ccpp-scm v6.0.0.0 manual is misleading

**Reply by dustinswales** [reply]
@AnningCheng-NOAA You bring up a good point. The UFS replay feature is on the "main" branch, not the v6.0.0 branch used in the v6.0.0 users guide. I will add a note in the current documentation mentioning this. Thanks for bringing this to our attention.
